cli/src/commands/contract/commands/update.ts

```diff
@@ -87,6 +90,30 @@ export default (opts: BaseCommandOptions) => {
         process.exit(1);
       }
     }
+
+    if (!options.suppressWarnings && resp.compositionWarnings.length > 0) {
```

Comment:
You could shorten this to `options.suppressWarnings?.length > 0` if you wanted.

Reply:
No, suppress warnings is a bool and composition warnings are from controlplane

Reply:
I'm an idiot.